static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -370,6 +380,11 @@ CStudioAuthoring.Module.requireModule(
 
             external_plugins: external,
 
+            style_formats: styleFormats,
+
+            // default to merge if no value is set
+            style_formats_merge: (styleFormats.length === 0 && styleFormatsMerge === null) || styleFormatsMerge,
```

Comment:
This is a change in current default behaviours. Why default it to try if nothing is set? The safe default seems to be `false`, which is [tinymce's default](https://www.tiny.cloud/docs/configure/editor-appearance/#style_formats_merge).

Are we trying to change the default for every body? @sumerjabri @russdanner 

A bit cleaner to initialize the value —with all/whatever logic above— and here just assign the prop.
```suggestion
            style_formats_merge: styleFormatsMerge,
```

Reply:
@rart We don't want to change the default. Actually, it was the opposite, I want to keep the default.
In the previous source code, we have `style_formats` default to `[]`. This caused the Format list to be empty and override TinyMCE default.

But I agreed that the code for `style_formats_merge` is not good. I changed back to the value as you suggested. And if there is no value set for `styleFormats`, the variable will be `undefined`. This should fix the default issue.